bin/concordance_validation/validate_and_delta.rb

```diff
@@ -25,9 +25,9 @@ def review_logs(log_file)
 
 conc_dir = Settings.concordance_path
 
-raw_gzip_files = Dir.glob("#{conc_dir}/raw/*txt.gz").sort
+raw_files = Dir.glob("#{conc_dir}/raw/*txt*").sort
```

Comment:
Any concerns about this matching e.g. "this_is_not_a_txt_file"? I figure we control what goes in that directory, so it shouldn't be a big issue?

Reply:
No more than it matching "junk_that_shouldnt_be_in_this_directory.txt.gz". If it gets garbage it will end up erroring out anyway.